README.md

```diff
@@ -135,18 +135,47 @@ The results of your analysis will be shown in the right pane.
 
 <details>
       <summary>Linux & macOS</summary>
-      
-The easiest way to run the program on Linux or macOS is by using the executable file located in the [releases](https://github.com/VerisimilitudeX/DNAnalyzer/releases/latest).
 
-First, download the zip or tar.gz file from releases and unzip it using the native utility. Then build gradle and run the GUI.
 
-```pwsh
+### DNAnalyzer & Java Download
+
+To run DNAnaylzer on Linux, you'll need to download the DNAnalyzer source code and download and install Java 17.
+
+First, download the zip or tar.gz file from [DNAnalyzer releases](https://github.com/VerisimilitudeX/DNAnalyzer/releases/latest) and unzip it using the native utility. 
```

Comment:
Your native utility

Reply:
yeah, like whatever zip manager you have, 7zip, gzip, tar